src/locales/ko/battle.ts

```diff
@@ -55,5 +55,7 @@ export const battle: SimpleTranslationEntries = {
   "notDisabled": "{{pokemonName}}의\n{{moveName}} 사슬묶기가 풀렸다!",
   "skipItemQuestion": "아이템을 받지 않고 넘어가시겠습니까?",
   "eggHatching": "어라…?",
-  "ivScannerUseQuestion": "{{pokemonName}}에게 개체값탐지기를 사용하시겠습니까?"
+  "ivScannerUseQuestion": "{{pokemonName}}에게 개체값탐지기를 사용하시겠습니까?",
+  "useSkill": "{{pokemonName}}(이)가 {{skillName}}을(를) 사용했다!",
+  "fainted": "{{pokemonName}}(이)가 쓰러졌다!"
```

Comment:
```suggestion
  "useSkill": "{{pokemonName}}의 {{skillName}}!",
  "fainted": "{{pokemonName}}(은)는 쓰러졌다!"
```

현재 최신 버전(9세대) 기준 배틀 메시지로 수정합니다.
modifying to the current korean version(9th gen) message

Reply:
제가 PR은 처음이라서 그러는데 이렇게 제안주신 코드를 적용하면 알아서 반영이 되는건가요?

Reply:
저도 직접 PR을 써 본적이 없어서 잘은 모르겠어요.  Files Changed를 보면 반영이 안 된 것 같은데, 직접 수정해 주셔야 하는 것 아닐까요?

Reply:
> 제가 PR은 처음이라서 그러는데 이렇게 제안주신 코드를 적용하면 알아서 반영이 되는건가요?

저도 직접 PR을 써 본적이 없어서 잘은 모르겠어요. Files Changed를 보면 반영이 안 된 것 같은데, 직접 수정해 주셔야 하는 것 아닐까요?

Reply:
PR에 쓰신 브랜치 `Ohmry:main`에 새로운 변경 사항을 커밋해 푸쉬해주시면 알아서 추가되더라구요

Reply:
아, 네네 다시 수정해서 푸시하도록 하겠습니다. 감사합니다.

Reply:
@MrWaterT @MrWaterT 추가로 [포켓로그 위키 디스코드](https://discord.gg/FPtbzYPJ) 채널에 위키 및 깃허브 관리자 분들도 계시던데, 여기에 인게임 번역 관련 방도 있어서 한국어 작업 하시는 분들이 여기 오시면 좀 더 작업이 편하지 않을까 싶습니다